tests/disas/pulley/pulley32_memory32.wat

```diff
@@ -0,0 +1,110 @@
+;;! target = "pulley32"
+;;! test = "compile"
+
+(module
+  (memory 1)
+
+  (func $load8 (param i32) (result i32)
+    (i32.load8_u (local.get 0)))
+
+  (func $load16 (param i32) (result i32)
+    (i32.load16_u (local.get 0)))
+
+  (func $load32 (param i32) (result i32)
+    (i32.load (local.get 0)))
+
+  (func $load64 (param i32) (result i64)
+    (i64.load (local.get 0)))
+
+  (func $load8_offset (param i32) (result i32)
+    (i32.load8_u offset=32 (local.get 0)))
+
+  (func $load16_offset (param i32) (result i32)
+    (i32.load16_u offset=32 (local.get 0)))
+
+  (func $load32_offset (param i32) (result i32)
+    (i32.load offset=32 (local.get 0)))
+
+  (func $load64_offset (param i32) (result i64)
+    (i64.load offset=32 (local.get 0)))
+)
+;; wasm[0]::function[0]::load8:
+;;       push_frame
+;;       xload32le_offset8 x6, x0, 52
+;;       br_if_xulteq32 x6, x2, 0x14    // target = 0x19
```

Comment:
We don't want this one to use the new `xbc` macro-op?

Reply:
No we probably do but I wanted to add this here for complete-ness to show that it's not currently using the new op. One-byte loads/stores are generated a bit differently with bounds checks and use a `>=` condition instead of `>` + arithmetic to test whether something is out of bounds. That made it so it doesn't fall into the patterns I added and would need a specific opcode just for the one-byte access. I figured it's probably fine to leave that off for now and add in the future if necessary